### DISCUSSION TITLE
Optimal settings for very flat prints?

### DISCUSSION THREAD

**Discussion by ogiesen** [FormerLurker/Octolapse#803] [General]
Has anyone found good settings for capturing a print that only has a few layers - let's say, fewer than 100, e.g. a plaque with a relief?

Taking one picture per layer would leave me with only a very short video that would not really convey much sense of progress. As far as I could tell that leaves me with the timer-based triggers where the downside is that a lot of the time the print head will be obscuring the print (I'm not a fan of moving the extruder away from the print to take pictures - for taller prints I prefer a Snap To Print trigger with Back Center stabilization)...

Would it somehow be possible to combine a stabilization option like "Back Center" with "Smart snap to print" functionality but still trigger based on time along the lines of "every x seconds take snapshot at the next suitable stabilization point"? (the time interval would be interpreted as a minimum time between shots rather than as a constant). As I can see that a time-based approach would probably be difficult, an alternative would be "after at least x mm travelled"...

Actually, I think an ideal stabilization for this type of print would actually be "whenever the extruder is at the maximum Y coordinate within the print for the current X"... (i.e. when it is the furthest away it could be from the camera without leaving the print)

What settings would you currently use for such a flat print (if you'd bother at all that is ;) )?

**Comment by FormerLurker** [comment]
You are describing a distance trigger (either mm extruded or mm traveled), which has been on my agenda for a long while.  Since that isn't implemented, I would have to recommend the timer trigger.  It does require some kind of stabilization, but I have gotten excellent results printing signs (a few mm thick) with lettering.  It is a really neat effect.  Stabilizing in the center will keep your quality high, or just outside of the camera's field of view.  Clever camera positioning can help immensely.  You can choose the exact coordinates via a custom stabilization profile.

I will start thinking about the distance trigger too.  Maybe it could make the next release.

**Reply by ogiesen** [reply]
Thanks, that sounds great!
Do you already know how/if that would interact with any stabilization options? Would/Could that act similar to a smart snap-to-print, choosing the stabilization point closest to the position where the distance triggered?
